### PR TITLE
[ticket/13992] Fix HTML5 error from w3.org validator

### PR DESCRIPTION
**Pull request by Zoddo**
https://tracker.phpbb.com/browse/PHPBB3-13992

### CODE REVIEW

**Comment by VSEphpbb** [comment]
Why are you getting rid of media=screen? The only invalid thing we had there was "projection" which we removed. screen is still very valid.

**Comment by Zoddo** [comment]
I removed these changes

File: phpBB/styles/prosilver/template/overall_header.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
 <!-- ENDIF -->
 <link href="{T_STYLESHEET_LINK}" rel="stylesheet">
 <link href="{T_STYLESHEET_LANG_LINK}" rel="stylesheet">
-<link href="{T_THEME_PATH}/responsive.css?assets_version={T_ASSETS_VERSION}" rel="stylesheet" media="all and (max-width: 700px), all and (max-device-width: 700px)">
+<link href="{T_THEME_PATH}/responsive.css?assets_version={T_ASSETS_VERSION}" rel="stylesheet" media="(max-width: 700px)">
```
Comment (VSEphpbb) [inline phpBB/styles/prosilver/template/overall_header.html]: I think it should be all and (max-width: 700px) ?

File: phpBB/styles/prosilver/template/simple_header.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
 <!-- ENDIF -->
 <link href="{T_STYLESHEET_LINK}" rel="stylesheet">
 <link href="{T_STYLESHEET_LANG_LINK}" rel="stylesheet">
-<link href="{T_THEME_PATH}/responsive.css?assets_version={T_ASSETS_VERSION}" rel="stylesheet" media="only screen and (max-width: 700px), only screen and (max-device-width: 700px)">
+<link href="{T_THEME_PATH}/responsive.css?assets_version={T_ASSETS_VERSION}" rel="stylesheet" media="(max-width: 700px)">
```
Comment (VSEphpbb) [inline phpBB/styles/prosilver/template/simple_header.html]: I think it should be only screen and (max-width: 700px) ?

**Pull request merged**